### PR TITLE
REPO-4745 update acs community packaging transformation tests

### PR DESCRIPTION
**Pull request by PunchedChimera**
Rendition tests jobs added to .travis.yml as in acs-packaging PR[#541](https://github.com/Alfresco/acs-packaging/pull/541).
Test job for Transform Service has been removed for community.

### CODE REVIEW

File: tests/environment/docker-compose-legacy-transforms-only.yml [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+                -Ddb.password=alfresco
+                -Ddb.url=jdbc:postgresql://postgres:5432/alfresco
+                -Dalfresco.restApi.basicAuthScheme=true
+                -Dmessaging.broker.url=\"failover:(tcp://activemq:61616)?timeout=3000&jms.useCompression=true\"
+                
+                -Dtransform.service.enabled=false
+
+                -Dlocal.transform.service.enabled=false
+
+                -Dlegacy.transform.service.enabled=true
```
Comment (killerboot) [inline tests/environment/docker-compose-legacy-transforms-only.yml]: Does it mean that you don't need the containers for the transformers as those are not going to be used?

Comment (PunchedChimera) [reply tests/environment/docker-compose-legacy-transforms-only.yml]: Tika and transform-misc containers were not required and have been removed.

Comment (killerboot) [reply tests/environment/docker-compose-legacy-transforms-only.yml]: Sorry, I'm a bit confused now :)
So,
* if we enable local transformers, this means that the executables in content-repository docker container are used to do transformations. In this case the URLs and docker containers for individual transform engines are not required.
* transform service does not exist in ACS Community, there is no such property like `transform.service.enabled` and tests for it should not be required.
* if you remove some of the transform engines (misc and tika) and tests still work, it means that the tests are incomplete or there is some configuration issue. I don't think that compose should have some engines missing, it's either all of them or none at all.

Comment (PunchedChimera) [reply tests/environment/docker-compose-legacy-transforms-only.yml]: The `transform.service.enabled` has been removed as this is not a community feature

Comment (PunchedChimera) [reply tests/environment/docker-compose-legacy-transforms-only.yml]: These are integration tests, intended to confirm that the legacy/ local/ or the Transform service, depending on which properties are turned on, can communicate. 
The Legacy and Local transform services are able to communicate with T-engines and these tests are to make sure this communication is working correctly, so these containers are needed for the tests. 
As you rightly said, the transform service is only for enterprise and has not been added.

**Pull request merged**